CppExport/src/exporter/CppExporter.cpp

```diff
@@ -56,7 +57,7 @@ QList<Export::ExportError> CppExporter::exportTree(Model::TreeManager* treeManag
 	for (auto codeComposite : mergeUnits(codeUnits))
 	{
 		codeComposite->sortUnits();
-		createFileFromFragment(directory, codeComposite->name() + ".h", codeComposite->headerFragment());
+		createFileFromFragment(directory, codeComposite->name() + ".h", addPragmaOnce(codeComposite->headerFragment()));
```

Comment:
Please move this stuff in two functions one in `CodeComposite`, the rest in `createFileFromFragment`